Updating Documentation around config status metrics (#8089)
solo-io/gloo: docs/content/guides/traffic_management/configuration_validation/_index.md

{{% /notice %}}

## Monitoring the configuration status of Gloo resources

> When Gloo Edge fails to process a resource, we see that reflected in the resource's {{< protobuf name="core.solo.io.Status" display="Status">}}.

Reviewer: ```suggestion
When Gloo Edge fails to process a resource, the failure is reflected in the resource's {{< protobuf name="core.solo.io.Status" display="Status">}}.
```

> You can configure Gloo Edge to publish metrics that record the configuration status of resources.

Reviewer: ```suggestion
For increased observability, you can also configure Gloo Edge to publish metrics that record the configuration status of resources.
```

In the `observabilityOptions` of the Settings CRD, you can enable status metrics by specifying the resource type and any labels to apply

> to the metric. Consider the following, which will add metrics for virtual services and
> upstreams, both having labels that include the namespace and name of each individual resource:

Reviewer: ```suggestion
to the metric. The following example adds metrics for virtual services and
upstreams, which both have labels that include the namespace and name of each individual resource:
```

```yaml
observabilityOptions:
  configStatusMetricLabels:
    Upstream.v1.gloo.solo.io:
      labelToPath:
        name: '{.metadata.name}'
        namespace: '{.metadata.namespace}'
    VirtualService.v1.gateway.solo.io:
      labelToPath:
        name: '{.metadata.name}'
        namespace: '{.metadata.namespace}'
```

> After completing the [Hello World guide]({{% versioned_link_path fromRoot="/guides/traffic_management/hello_world/" %}}) 
> to generate some resources, the metrics defined above can be found at: <http://localhost:9091/metrics>. If the port
> forwarding is directed towards the gloo pod, the `default-petstore-8080` upstream can be seen in a healthy state:

Reviewer: ```suggestion
After you complete the [Hello World guide]({{% versioned_link_path fromRoot="/guides/traffic_management/hello_world/" %}}) 
to generate some resources, you can see the metrics that you defined at <http://localhost:9091/metrics>. If the port
forwarding is directed towards the Gloo pod, the `default-petstore-8080` upstream reports a healthy state:
```
